src/content/docs/en/guides/deploy/zerops.mdx

```diff
@@ -0,0 +1,178 @@
+---
+title: Deploy your Astro Site to Zerops
+description: How to deploy your Astro site to the web using Zerops.
+type: deploy
+i18nReady: true
+---
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+[Zerops](https://zerops.io/) is a dev-first cloud platform that can be used to deploy an Astro site. As of April 2024, it only supports SSR deployments.
+
+This guide will walk you through deploying an Astro Node.js service to Zerops.
+
+## Prerequisites
+
+- A Zerops account. If you don’t already have one, you can create an account for free.
+- `@astrojs/node` package
+
+:::tip[How about a one-click example?]
+Import [Zerops x Astro - Node.js](https://github.com/zeropsio/recipe-astro-nodejs) example app into Zerops.
+Open [Zerops Dashboard](https://app.zerops.io/dashboard/projects), click "Import project", paste ⬇️ — built, deployed, done.
+```yaml
+project:
+  name: astro
+services:
+  - hostname: astronode
+    type: nodejs@20
+    buildFromGit: https://github.com/zeropsio/recipe-astro-nodejs
+    ports:
+      - port: 4321
+        httpSupport: true
+    enableSubdomainAccess: true
+    minContainers: 1
+```
+:::
+
+### Creating your own Zerops project with Astro apps
+
+You can either add project and your first Node.js service through the [project add](https://app.zerops.io/dashboard/project-add) wizard in Zerops app, or quickly by importing it using a yaml structure.
+
+```yaml
+project:
+  name: my-astro-sites
+services:
+  - hostname: hellothere
+    type: nodejs@20
+    ports:
+      - port: 4321
+        httpSupport: true
+    minContainers: 1
+```
+
+This will setup a project called `my-astro-sites`, with a Node.js v20 service called `hellothere`, which will be ready for you to deploy your Astro app into. One Zerops project can contain many Astro apps.
+
+### Setting Up Node.js SSR in your app
+Zerops is using [`@astrojs/node`](/en/guides/integrations-guide/node/) package to run the app. Add it to your deps, if you haven't already, and modify your Astro config to utilize it.
+
+Here are the necessary lines in your `astro.config.mjs` file:
+```js title="astro.config.mjs"
+  import { defineConfig } from 'astro/config';
+  import node from "@astrojs/node";
+
+  export default defineConfig({
+    output: 'server',
+    adapter: node({
+      mode: "standalone"
+    }),
+  });
+```
+
```

Comment:
```suggestion
```
This was in prerequisites, so we don't need it here! We can assume it's already done.